src/platform/utilities/cerner/index.js

```diff
@@ -2,6 +2,7 @@
 import environment from 'platform/utilities/environment';
 
 export const CERNER_FACILITY_IDS = ['668', '757'];
+export const isCernerLive = !environment.isProduction();
```

Comment:
This probably makes sense as a feature flag, but I think that's complicated with the website content work.